File: Blueprism/BluePrism.Api/BluePrism.Api.Domain/Dashboard/WorkQueueComposition.cs
namespace BluePrism.Api.Domain.Dashboard
{
    using System;

    public class WorkQueueComposition
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public int Completed { get; set; }
        public int Pending { get; set; }
        public int Deferred { get; set; }
        public int Locked { get; set; }
        public int Exceptioned { get; set; }
    }
}

File: Blueprism/BluePrism.Server.Domain.Models/Dashboard/WorkQueueComposition.vb
Imports System.Runtime.Serialization

Namespace Dashboard

    <Serializable>
    <DataContract([Namespace]:="bp")>
    Public Class WorkQueueComposition
        <IgnoreDataMember>
        Public Id As Guid
        <IgnoreDataMember>
        Public Name As String
        <IgnoreDataMember>
        Public Completed As Integer
        <IgnoreDataMember>
        Public Pending As Integer
        <IgnoreDataMember>
        Public Deferred As Integer
        <IgnoreDataMember>
        Public Locked As Integer
        <IgnoreDataMember>
        Public Exceptioned As Integer
    End Class
End NameSpace
